plugins/modules/opensearch.py

```diff
@@ -0,0 +1,1194 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright: Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+from __future__ import absolute_import, division, print_function
+
+DOCUMENTATION = """
+---
+module: opensearch
+short_description: Creates OpenSearch or ElasticSearch domain.
+description:
+  - Creates a new Amazon OpenSearch Service domain.
+version_added: 3.0.0
```

Comment:
Since community.aws 3.0.0 has already been released, you can update `version_added: 3.1.0` for now.

Reply:
done